include/grass/gis.h

```diff
@@ -154,6 +154,12 @@ static const char *GRASS_copyright UNUSED = "GRASS GNU GPL licensed Software";
 #define MIN(a, b) ((a) < (b) ? (a) : (b))
 
 /* define PI and friends */
+#if defined(_MSC_VER)
+#define _USE_MATH_DEFINES 1
+#include <math.h>
+#undef min
+#undef max
+#else
```

Comment:
Is this better than the changes done by #1795, to always use our MIN and MAX? (I think what we have now is better than the solution implemented and shown here)
```suggestion
```